.github/workflows/reusable-e2e-tests-build.yml

```
@@ -6,7 +6,9 @@ on:
 jobs:
   e2e-tests-build:
     name: 'Tests: End-to-end (build job)'
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
+    container:
+      image: cypress/browsers:node-20.13.1-chrome-125.0.6422.60-1-ff-126.0-edge-125.0.2535.51-1
```

```suggestion
    runs-on: ubuntu-22.04
```

To build the e2e images, we don't want the cypress image.
We want access to an environment which can build OCI (Open Container Initiative) images fast.